Client/Extensions/NSDirectionalEdgeInsets+Extension.swift

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension NSDirectionalEdgeInsets {
```

Comment:
As mentioned above we are trying to [move away from using extensions](https://github.com/mozilla-mobile/firefox-ios/wiki/When-To-Use-Extensions). Could you please remove this one?

Reply:
Done 🙂